src/test/java/seedu/address/logic/candidate/CommandTestUtil.java

```diff
@@ -86,6 +91,8 @@ public static void assertCommandSuccess(Command command, Model actualModel, Comm
                                             Model expectedModel) {
         try {
             CommandResult result = command.execute(actualModel);
+            System.out.println(expectedCommandResult.getFeedbackToUser());
+            System.out.println(result.getFeedbackToUser());
```

Comment:
Are these 2 println statement part of the assertCommandSuccess method? What are their purposes?